src/components/skip-link/_skip-link.scss

```diff
@@ -0,0 +1,40 @@
+@import "../../globals/scss/import-once";
+@import "../../globals/scss/font-face";
+@import "../../globals/scss/typography";
+@import "../../globals/scss/links";
+@import "../../globals/scss/vars";
+
+@include exports("skip-link") {
+  .govuk-c-skip-link {
+    background: $govuk-black;
+    text-align: center;
+  }
+
+  .govuk-c-skip-link__content-wrapper {
+    max-width: 1020px;
```

Comment:
I'm a little confused as to what this is meant to do – it sets a max width but because the link itself is positioned absolutely I don't think this actually does anything?